conanfile.py

```diff
@@ -39,6 +39,7 @@ def requirements(self):
         self.requires("dice-hash/0.4.11", transitive_headers=True)
         self.requires("dice-sparse-map/0.2.9", transitive_headers=True)
         self.requires("dice-template-library/1.19.0", transitive_headers=True)
+        self.requires("libxml2/2.15.0", options={"iconv": False})
```

Comment:
is there a difference between specifiying it here and in the `configure()` step?